mysql-test/suite/galera/t/pxc_drop_user.test

```diff
@@ -0,0 +1,115 @@
+# Test that DROP USER executed on the user with SYSTEM_USER privilege
+# is not replicated when executing user has no SYSTEM_USER
+
+--source include/galera_cluster.inc
+--source include/count_sessions.inc
```

Comment:
This also needs have_debug_sync with the latest changes.

Reply:
added